source/elements/oneDPL/source/parallel_api/parallel_range_api.rst

```diff
@@ -46,9 +47,31 @@ of parallel range algorithms.
 .. code:: cpp
 
    // C++20 analogue of std::projected_value_t; exposition only
-   template<typename I, typename Proj>
+   template <typename I, typename Proj>
    using /*projected-value-type*/ = std::remove_cvref_t<std::invoke_result_t<Proj&, std::iter_value_t<I>&>>;
 
+  // C++20 analogue of nothrow-random-access-range proposed for C++26 in P3179R9; exposition only
+  template <typename R>
+  concept nothrow-random-access-range =
```

Comment:
why did we decide to "copy" this concept instead of *`nothrow-sized-random-access-range`*?